web/libs/editor/src/components/Comments/Comment/CommentForm.tsx

```diff
@@ -114,36 +118,69 @@ export const CommentForm: FC<CommentFormProps> = observer(({ commentStore, annot
 
   const currentLinkingComment = annotationStore.selected.currentLinkingMode?.comment;
   const currentComment = getCurrentComment();
-  const { text = "", regionRef } = currentComment || {};
+  const { text = "", regionRef, classifications } = currentComment || {};
   const { region, result } = regionRef || {};
   const linking = !!linkingComment && currentLinkingComment === linkingComment && globalLinking;
   const hasLinkState = linking || region;
+  const selections = useMemo(() => taxonomyPathsToSelectedItems(classifications?.default?.values), [classifications]);
+  const classificationsItems = commentStore.commentClassificationsItems;
+
+  const updateCommentClassifications = useCallback(
+    (classifications: object | null) => {
+      const currentComment = getCurrentComment();
+      currentComment.setClassifications(classifications);
+    },
+    [commentStore, annotationStore],
```

Comment:
```suggestion
    [getCurrentComment],
```
I didn't see this one earlier, but if the only reason this function has to change is if the other function changes, it is the only dependency required here. commentStore and annotationStore are not used within this function, only getCurrentComment.